templates/rhel75-hp.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: rhel-high-performance
+  annotations:
+    openshift.io/display-name: "Red Hat Enterprise Linux 7.0+ VM High Performance"
+    description: >-
+      This template can be used to create a VM suitable for
+      Red Hat Enterprise Linux 7 and newer and sets configuration
+      for high performance. That means for example CPU passhtrough,
+      CPU pinning and disabled graphics device.
+      The template assumes that a PVC is available which is providing the
+      necessary RHEL disk image.
+    tags: "kubevirt,virtualmachine,linux,rhel,high-performance"
+    iconClass: "icon-rhel"
+    openshift.io/provider-display-name: "KubeVirt"
+    openshift.io/documentation-url: "https://github.com/fabiand/common-templates"
+    openshift.io/support-url: "https://github.com/fabiand/common-templates/issues"
+    template.openshift.io/bindable: "false"
+
+    defaults.template.cnv.io/disk: rootdisk
+    template.cnv.io/editable:
+    - "/objects[0].spec.template.spec.domain.cpu.cores"
+    - "/objects[0].spec.template.spec.domain.resources.requests.memory"
+    - "/objects[0].spec.template.spec.domain.devices.disks"
+    - "/objects[0].spec.template.spec.volumes"
+    - "/objects[0].spec.template.spec.networks"
+
+  labels:
+    os.template.cnv.io/rhel7.0: true
+    os.template.cnv.io/rhel7.1: true
+    os.template.cnv.io/rhel7.2: true
+    os.template.cnv.io/rhel7.3: true
+    os.template.cnv.io/rhel7.4: true
+    os.template.cnv.io/rhel7.5: true
+    workload.template.cnv.io/high-performance: true
+    flavor.template.cnv.io/medium: true
+
+objects:
+- apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    running: false
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: 2
+            dedicatedCpuPlacement: true
+            model: host-model
+          resources:
+            requests:
+              memory: 4G
```

Comment:
Here again, I do believe that in order to help us in the future we should at least have a nice comment on why we decided to go for 4GB instead of something else.

Reply:
And the comment also applies for the number of cores.